src/components/WhatsNewLayout.tsx

```diff
@@ -1,13 +1,35 @@
+import {
+    AnchorHTMLAttributes,
+    BlockquoteHTMLAttributes,
+    HTMLAttributes,
+    ImgHTMLAttributes,
+    LiHTMLAttributes,
+    OlHTMLAttributes,
+} from 'react';
 import { MDXProvider } from '@mdx-js/react';
 import Head from 'next/head';
-import { Md } from '@taskany/bricks/harmony';
 import { useTheme } from 'next-themes';
+import { Link, Image, Text } from '@taskany/bricks/harmony';
 
 interface WhatsNewLayoutProps {
     children: React.ReactNode;
 }
 
-const components = {};
+const components = {
+    a: (props: AnchorHTMLAttributes<HTMLAnchorElement>) => <Link {...props} target="_blank" view="inline" />,
```

Comment:
Could you use single components list for both components (whatsnew/md) with in-place overrides?